Mathlib/AlgebraicTopology/SimplicialObject/Basic.lean

```diff
@@ -264,6 +264,11 @@ scoped macro_rules
     `(($X : CategoryTheory.SimplicialObject.Truncated _ $n).obj
       (Opposite.op ⟨SimplexCategory.mk $m, $p⟩))
 
+variable (C) in
+/-- Further truncation of truncated simplicial objects. -/
+def trunc (n m : ℕ) (h : m ≤ n := by omega) : Truncated C n ⥤ Truncated C m :=
```

Comment:
We probably want some `simp` lemmas attached to this definition.

Reply:
I have added the `@[simps!]` attribute here, but please let me know if you had some more specific simp lemmas in mind.